keyboards/adkb96/keymaps/default/keymap.c

```diff
@@ -0,0 +1,15 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+const uint16_t PROGMEM keymaps[1][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
You don't need to specify the number of layers; the compiler can figure this out on its own, and it becomes another thing to look out for if you want to add more layers later on.
```suggestion
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Reply:
Thank you,I fixed it!